config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-  root 'application#hello'
+  root 'login#login'
+  get '/home', to: 'home#home'
+  get '/register', to: 'users#new'
+  post '/register',  to: 'users#create'
+  get '/login', to: 'login#login'
```

Comment:
im not sure how this works when the routes for login are:
`get '/login', to: 'login#login'`
`post '/login', to: 'login#create'`
`delete '/logout', to: 'login#destroy'`
Is
`resources :login, only: [:index, :new, :create, :destroy]`
an alias for this?
`index '/login', to: 'login#index'`
`get '/login', to: 'login#new'`
`post '/login', to: 'login#create'`
`delete '/login', to: 'login#destroy'`